CONTRIBUTING.es.md

```diff
@@ -0,0 +1,176 @@
+  published: '2023-04-11',
+  tags: ["meta"],
+  license: 'cc-by-4'
+}
+---
+
+¡Hola! ¡Ésta es mi primera entrada! (TODO: escribir más texto aquí)
+```
+
+> **Nota**: El título que definas en el campo "title" siempre se verá en la parte superior de tu entrada. No necesitas iniciar tu entrada con otro encabezado - de lo contrario ¡tu entrada tendrá dos títulos!
+
+<details>
+  <summary><strong>Propiedades opcionales</strong></summary>
+
+  Existen algunas propiedades extra que *podrías* incluir en el frontmatter de tu entrada, pero no son necesarias:
+
+  - `authors: ["autor1", "autor2"]` se puede usar para especificar de forma manual los ID's de los autores de una entrada, esto cuando la publicación tenga más de un autor.
+  - `edited: "2023-10-21"` sirve para especificar la fecha en la hiciste la "última actuaización" de tu entrada en caso de que realices modificaciones.
+  - `collection: "Mi genial serie de artículos"` tratará a un grupo de publicaciones como una serie en caso de que todas tengan la misma cadena `collection` configurada.
+  - `order: 0` reordenará las entradas de una colección de acuerdo con el valor que proporciones. Esto no tendrá efecto a menos que la entrada se encuentre dentro de una colección.
+  - `originalLink: "https://example.com"` especifica una URL externa que sirva como fuente para tu entrada. ¡Es importante especificar este valor si estás republicando algo que tengas escrito en otro blog!
+
+</details>
+
+## Licencias
+
+Proporcionar una licencia ayuda a explicar lo que los lectores pueden hacer con su trabajo - ya sea que puedan usarlo como material para un curso, o reusarlo en otras formas. Visita [el sitio de Creative Commons](https://creativecommons.org/about/cclicenses/) para obtener una visión general de lo que se permite hacer dentro de los distintos tipos de licencias.
+
+Actualmente, estas son las licencias de creative commons que se permiten como valores dentro de la propiedad "license":
+
+- [`'cc-by-4'`](http://creativecommons.org/licenses/by/4.0/)
+- [`'cc-by-nc-sa-4'`](http://creativecommons.org/licenses/by-nc-sa/4.0/)
+- [`'cc-by-nc-nd-4'`](https://creativecommons.org/licenses/by-nc-nd/4.0/)
+- [`'publicdomain-zero-1'`](https://creativecommons.org/publicdomain/zero/1.0/)
+
+También puedes omitir la propiedad "license". En este caso, tu entrada quedará bajo la licencia [MPL 2.0](https://github.com/playfulprogramming/playfulprogramming/blob/main/LICENSE.md) del repositorio.
+
+## Enlaces incrustados
+
+Las entradas pueden incristar sus propias etiquetas `<iframe>` si es necesario - éstas mostrarán una vista previa de "haz clic para ejecutar" y no afectarán el tiempo de carga de la página.
+
+También puedes incrustar algunos servicios de terceros simplemente pegando el enlace en tu entrada, como vídeos de YouTube o de Twitch, publicaciones de Twitter - y cualquier cosas compatible con [oembed.com](https://oembed.com).
+
+## Imágenes y vídeos
```

Comment:
There's a "dilemma" here ... `vídeo` is used in Spain, while `video` is used in most of other Spanish-speaking countries 🤔 

```suggestion
## Imágenes y videos
```